generators/app/templates/_package.json

```diff
@@ -28,19 +28,13 @@
     "generator-jhipster": ">2.26.0"
   },
   "devDependencies": {
-    "gulp": "^3.9.0",
```

Comment:
Oh you're right. Sorry, I didn't have a closed look to your last PR and the new file .gulpfile.js !
I will fix that!